content/python/concepts/files/terms/unlink/unlink.md

```diff
@@ -40,3 +40,24 @@ import os
 # Delete file path
 os.unlink("path/to/my_file.txt")
 ```
+## Codebyte Example
+
+In this example **delete_me.txt** is deleted using the complete file path and the `.unlink()` method:
+
+```codebyte/python
+import os
+
+#Create file
+f = open("/tmp/delete_me.txt", "w")
+f.write("Bye bye!")
+f.close()
+
+#List files in directory
+print("Files in tmp directory are: ", os.listdir("/tmp"))
```

Comment:
```suggestion
print("Files in the current directory are: ", os.listdir("/home/ccuser"))
```

Reply:
Hi @caupolicandiaz  because we are changing the directory here to "/home/ccuser" I just want to confirm I will also need to update line 51 as well so that delete_me.txt will be created in the correct location?

Thanks!